tensorboard/plugins/mesh/metadata_test.py

```diff
@@ -75,9 +82,10 @@ def test_parse_plugin_metadata(self):
 
   def test_metadata_version(self):
     """Tests that only the latest version of metadata is supported."""
-    self._create_metadata()
-    # Change the version.
     with patch.object(metadata, 'get_current_version', return_value=100):
+      self._create_metadata()
+    # Change the version.
+    with patch.object(metadata, 'get_current_version', return_value=1):
```

Comment:
Why are we mocking this at all, much less to a return value that’s
larger than any known plugin version? Shouldn’t this cause the check on line 121 of `parse_plugin_metadata` to fail?

```python
  if not 0 <= result.version <= get_current_version():
    raise ValueError(...)
```